ui/tests/integration/components/secret-engine/configure-wif-test.js

```diff
@@ -0,0 +1,901 @@
+              `);
+          this.server.post(configUrl('aws', this.id), () => {
+            assert.true(
+              false,
+              'post request was made to config/root when user canceled out of flow. test should fail.'
+            );
+          });
+          this.server.post(configUrl('aws-lease', this.id), () => {
+            assert.true(
+              false,
+              'post request was made to config/lease when user canceled out of flow. test should fail.'
+            );
+          });
+          this.server.post('/identity/oidc/config', () => {
+            assert.true(
+              false,
+              'post request was made to save issuer when user canceled out of flow. test should fail.'
+            );
+          });
+          // fill in both lease and root endpoints to ensure that both payloads are attempted to be sent
+          await fillInAwsConfig('withWif');
+          await fillInAwsConfig('withLease');
+          await click(GENERAL.cancelButton);
+
+          assert.true(this.flashDangerSpy.notCalled, 'No danger flash messages called.');
+          assert.true(this.flashSuccessSpy.notCalled, 'No success flash messages called.');
+          assert.ok(
+            this.transitionStub.calledWith('vault.cluster.secrets.backend.configuration', this.id),
+            'Transitioned to the configuration index route.'
+          );
+        });
+      });
+
+      module('Issuer field tests', function () {
+        test('if issuer API error and user changes issuer value, shows specific warning message', async function (assert) {
+          this.id = `azure-${this.uid}`;
+          this.displayName = 'Azure';
+          this.issuerConfig = createConfig(this.store, this.id, 'issuer');
+          this.issuerConfig.queryIssuerError = true;
+          this.config = this.store.createRecord('azure/config');
+          this.config.backend = this.id;
+
+          await render(hbs`
+                <SecretEngine::ConfigureWif @backendPath={{this.id}} @displayName={{this.displayName}} @type='azure' @model={{this.config}} @issuerConfig={{this.issuerConfig}}/>
+              `);
+          await click(SES.wif.accessType('wif'));
+          await fillIn(GENERAL.inputByAttr('issuer'), 'http://change.me.no.read');
+          await click(GENERAL.saveButton);
+          assert
+            .dom(SES.wif.issuerWarningMessage)
+            .hasText(
+              `You are updating the global issuer config. This will overwrite Vault's current issuer if it exists and may affect other configurations using this value. Continue?`,
+              'modal shows message about overwriting value if it exists'
+            );
+        });
+
+        test('it shows placeholder issuer, and does not call APIs on canceling out of issuer modal', async function (assert) {
+          this.server.post('/identity/oidc/config', () => {
+            assert.notOk(true, 'request should not be made to issuer config endpoint');
```

Comment:
can we use `assert.false` instead? and same question about throwing the error instead 🙃